src/theme/utils.ts

```diff
@@ -35,17 +35,18 @@ export const colorShadesCreator = (base: string, per: number) => {
     }, {});
 };
 
-export const iterateObject = (obj: any, func: (base: string, per: number) => any) =>
-  Object.keys(obj).reduce((acc, value, arr) => {
+export const iterateObject = (obj: any, func: (obj: any, value: any) => any) =>
```

Comment:
Can we have a little stricter types here? And maybe a type for the return value?

Reply:
On the iterateObject I left it with any because I thought we might need it in something other than the palette. Will change it though and we can revisit this in the future.